supervision/detection/core.py

```diff
@@ -324,6 +326,28 @@ def from_roboflow(cls, roboflow_result: dict, class_list: List[str]) -> Detectio
             class_id.append(class_list.index(prediction["class"]))
             confidence.append(prediction["confidence"])
 
+            if prediction.get("points"):
+                points = prediction["points"]
+
+                mask = np.zeros((int(height), int(width)), dtype=np.uint8)
```

Comment:
Most of that logic can be replaced with `polygon_to_mask` function that can create a mask given `polygon`. Please use it. You can find it in `supervision.detection.utils`.